automol/util/ring.py

```diff
@@ -174,16 +151,15 @@ def cycle_items_to_back(items: List[object], back_items: List[object]) -> List[o
 
 
 def cycle_to_split(
-    items: List[object], split_pair: Tuple[object, object]
-) -> List[object]:
-    """Cycle to split a pair of adjacent items, putting one on each end of the list
+    items: list[object], split_pair: tuple[object, object]
+) -> list[object]:
```

Comment:
See above. In this case, though, keep `split_pair` as a tuple because I'm not sure otherwise how to indicate that it must have exactly two elements.